core/store/ledgerstore/ledger_store.go

```diff
@@ -584,8 +628,15 @@ func (this *LedgerStoreImp) AddBlock(block *types.Block, stateMerkleRoot common.
 	if err != nil {
 		return fmt.Errorf("verifyHeader error %s", err)
 	}
-
-	err = this.saveBlock(block, stateMerkleRoot)
+	if ccMsg != nil {
+		if ccMsg.Height != currBlockHeight {
```

Comment:
should compare `ccMsg.Height ` with `block.Header.Height`.

Reply:
it's no problem, when you consensus block height = 105, currBlockHeight = 104, ccMsg.Height = 104.

Reply:
it's no problem.